ui/app/services/keyboard.js

```diff
@@ -42,122 +66,121 @@ export default class KeyboardService extends Service {
+        rebindable: true,
       },
-      requireModifier: true,
-    },
-    {
-      label: 'Previous Subnav',
-      pattern: ['Shift+ArrowLeft'],
-      action: () => {
-        this.traverseLinkList(this.subnavLinks, -1);
+      {
+        label: 'Go to Servers',
+        action: () => this.router.transitionTo('servers'),
+        rebindable: true,
       },
-      requireModifier: true,
-    },
-    {
-      label: 'Previous Main Section',
-      pattern: ['Shift+ArrowUp'],
-      action: () => {
-        this.traverseLinkList(this.navLinks, -1);
+      {
+        label: 'Go to Clients',
+        action: () => this.router.transitionTo('clients'),
+        rebindable: true,
       },
-      requireModifier: true,
-    },
-    {
-      label: 'Next Main Section',
-      pattern: ['Shift+ArrowDown'],
-      action: () => {
-        this.traverseLinkList(this.navLinks, 1);
+      {
+        label: 'Go to Topology',
+        action: () => this.router.transitionTo('topology'),
+        rebindable: true,
       },
-      requireModifier: true,
-    },
-    {
-      label: 'Show Keyboard Shortcuts',
-      pattern: ['Shift+?'],
-      action: () => {
-        this.shortcutsVisible = true;
+      {
+        label: 'Go to Evaluations',
+        action: () => this.router.transitionTo('evaluations'),
+        rebindable: true,
       },
-    },
-    {
-      label: 'Konami',
-      pattern: [
-        'ArrowUp',
-        'ArrowUp',
-        'ArrowDown',
-        'ArrowDown',
-        'ArrowLeft',
-        'ArrowRight',
-        'ArrowLeft',
-        'ArrowRight',
-        'b',
-        'a',
-      ],
-      action: () => {
-        console.log('Extra Lives +30');
+      {
+        label: 'Go to ACL Tokens',
+        action: () => this.router.transitionTo('settings.tokens'),
+        rebindable: true,
       },
-    },
-  ]);
+      {
+        label: 'Next Subnav',
+        action: () => {
+          this.traverseLinkList(this.subnavLinks, 1);
+        },
+        requireModifier: true,
+        rebindable: true,
+      },
+      {
+        label: 'Previous Subnav',
+        action: () => {
+          this.traverseLinkList(this.subnavLinks, -1);
+        },
+        requireModifier: true,
+        rebindable: true,
+      },
+      {
+        label: 'Previous Main Section',
+        action: () => {
+          this.traverseLinkList(this.navLinks, -1);
+        },
+        requireModifier: true,
+        rebindable: true,
+      },
+      {
+        label: 'Next Main Section',
+        action: () => {
+          this.traverseLinkList(this.navLinks, 1);
+        },
+        requireModifier: true,
+        rebindable: true,
+      },
+      {
+        label: 'Show Keyboard Shortcuts',
+        action: () => {
+          this.shortcutsVisible = true;
+        },
+      },
+    ].map((command) => {
+      const persistedValue = window.localStorage.getItem(
+        `keyboard.command.${command.label}`
+      );
```

Comment:
Dealing with localStorage directly in the service is the result of trying to use the utils/local-storage computed for awhile to no avail. This gives more control.

Reply:
Yup, I noticed that in the `action` for `enable` you're using, as well. I think you might want to read how the `utils/local-storage` works.

I think because `localStorageProperty` is a `computed` property -- which always makes me think that the behavior isn't synchronous which is a problem inside of a callback in a `map` function.